.github/CODEOWNERS

```
@@ -1,8 +1,10 @@
 /docs/ @apollographql/docs
 /.changesets/ @apollographql/docs
 /apollo-federation/ @dariuszkuc @sachindshinde @goto-bus-stop @SimonSapin @lrlna @TylerBloom @duckki
+/apollo-federation/src/source/connect @apollographql/connectors @nicholascioli
```

```suggestion
/apollo-federation/src/sources/connect @apollographql/connectors @nicholascioli
```

Does the line below this override ours? Or is it additive? I think we want to _also_ get pinged for reviews on `json_selection`

---

i dunno, but no harm in being redundant!